benchmarks/benchmarks/bench_lib.py

```diff
@@ -13,16 +13,28 @@ class Pad(Benchmark):
 
     param_names = ["shape", "pad_width", "mode"]
     params = [
-        [(1000,), (10, 100), (10, 10, 10)],
-        [1, 3, (0, 5)],
-        ["constant", "edge", "linear_ramp", "mean", "reflect", "wrap"],
+        [(1000,),
+         (10, 100),
+         (10, 10, 10),
+         # 50 * 512 * 512 = 13 million points = 46 MB. should be a good
```

Comment:
What kind of dtype takes 3.5 bytes? 
```python
>>> 46 / (50 * 512 * 512 / 1024 / 1024)                                           
3.68
```

Reply:
Actually that is something you added in 605fc1d7f871b1c8cf2783f594dee88cdf96d7ec. I have to confess that I just went with it.

Reply:
Haha, I guess I was crazy.